spec/models/home_controller_spec.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe AllArticlesController, type: :controller do
+  describe 'get index' do
```

Comment:
По правилам, `describe` должен описывать либо класс, либо его методы. Подробнее можно почитать тут https://www.betterspecs.org/#describe
```suggestion
  describe '#index' do
```